src/binary-reversal/index.js

```diff
@@ -3,6 +3,31 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+
+/**Stepts to binary reversal
+ * convert the string value into binary using toString method
+ * pad the conversion to 8 bits
+ * reverse the 8-bit conversion using while loop
+ * use regex to match one or more of the binary group of 01k
```

Comment:
**Noted**- I Would have done `.split('').reverse().join(''). `

This really would have made the code more comprehensive and there would have been no need for a `regex `after all

_I initially used regex because after using the `while loop`, it had undefined concatenated with the binary values so i just needed only the 01's_

Thank you